openedx/core/djangoapps/content/learning_sequences/api/processors/schedule.py

```diff
@@ -75,6 +75,13 @@ def inaccessible_sequences(self, full_course_outline):
                     seq_start = self.keys_to_schedule_fields[seq.usage_key].get('start')
                     if seq_start and self.at_time < seq_start:
                         inaccessible.add(seq.usage_key)
+                        continue
+
+                    seq_due = self.keys_to_schedule_fields[seq.usage_key].get('due')
```

Comment:
@cpennington, @Dillon-Dumesnil: If someone specifies the `due` attribute at a higher level in OLX (the chapter or top level course descriptor), will edx-when return the value properly for the sequence?

I'm not saying I necessarily want to support that non-standard use case–just want to be intentional if we're breaking it now.

Reply:
Based on what I'm seeing in the transformer that sets due dates (https://github.com/edx/edx-when/blob/master/edx_when/field_data.py#L124), we simply return all of the dates we have stored in edx-when. When we extract the due dates (https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/course_date_signals/handlers.py#L41), I believe for instructor paced courses, we will only take dates explicitly set by the instructor and for our Personalized Learner Schedule Dates, we start at the Section level with an offset and set it on the section and all of its children.

So in short, I _think_ that no, we would not as the due date would be in an instructor paced course (or we would ignore it in a self-paced) and it looks like we only send the date for the block it is on in that case and don't try to propagate it down. Hopefully @cpennington can confirm what I'm saying, but I think that's correct.

Reply:
Got it. Thank you.